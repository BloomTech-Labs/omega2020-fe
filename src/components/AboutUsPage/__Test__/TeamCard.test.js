import React from 'react';
import { render } from '@testing-library/react';
import  { configure, shallow, mount } from 'enzyme';
import '@testing-library/jest-dom/extend-expect';
import List from '../TeamCard';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

import Box from '@material-ui/core/Box';
import Avatar from '@material-ui/core/Avatar';
import Typography from '@material-ui/core/Typography';
import IconButton from '@material-ui/core/IconButton';
import GitHubIcon from '@material-ui/icons/GitHub';
import LinkedInIcon from '@material-ui/icons/LinkedIn';
import ClickAwayListener from '@material-ui/core/ClickAwayListener';
import Grow from '@material-ui/core/Grow';
import Popper from '@material-ui/core/Popper';
import MenuList from '@material-ui/core/MenuList';

configure({ adapter: new Adapter() });

describe('the components', () => {

    let wrapper;
        beforeEach(() => { wrapper = shallow(<List />)});
        
        it.skip('renders the snapshot', () => {
            const wrapper = renderer.create(<List />).toJSON();
            expect(wrapper).toMatchSnapshot();
        })
        
        it.skip('check if the compnonent exist', () => {
        expect(wrapper.exists()).toBe(true);
        });


        it('checks the length of the material ui', () => {
            const wrapper = mount(<List />)
            expect(wrapper.find(Box)).toHaveLength(2)
            expect(wrapper.find(Avatar)).toHaveLength(1)
            expect(wrapper.find(Popper)).toHaveLength(1)
        })
      
})

