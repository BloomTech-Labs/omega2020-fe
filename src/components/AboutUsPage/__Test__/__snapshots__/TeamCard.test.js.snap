// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the components renders the snapshot 1`] = `
<div
  className="MuiBox-root MuiBox-root-4 makeStyles-root-1"
>
  <div
    aria-haspopup="true"
    className="MuiAvatar-root MuiAvatar-circle makeStyles-large-2 MuiAvatar-colorDefault"
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
      }
    }
  >
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root MuiAvatar-fallback"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
      />
    </svg>
  </div>
  <br />
  <div
    className="MuiBox-root MuiBox-root-5"
  />
</div>
`;
